ci/test_wheel.sh

```
@@ -15,6 +15,17 @@ mkdir -p "${RAPIDS_TESTS_DIR}"
 # dependencies that can be installed later on when installing the wheel
 rapids-generate-pip-constraints test_python "${PIP_CONSTRAINT}"
 
+python -m venv libnvforest-env
+. libnvforest-env/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: libnvforest-env/bin/activate was not specified as input (see shellcheck -x).

(SC1091)